pkg/test/utils/kubernetes.go

```diff
@@ -1004,7 +998,7 @@ func RunCommands(logger Logger, namespace string, command string, commands []har
 	}
 
 	for _, cmd := range commands {
-		logger.Logf("running command: %s %s", command, cmd)
+		logger.Logf("running command: %s %q", command, cmd.Command)
```

Comment:
output for other reviewers

```
kuttl/harness/cli-test: logger.go:41: 16:24:57 | cli-test/1-patch | running command:  "kubectl label pod cli-test-pod test=true"
```

vs what used to be:

```
kuttl/harness/cli-test: logger.go:41: 16:33:29 | cli-test/1-patch | running command:  {"kubectl label pod cli-test-pod test=true" %!q(bool=true) %!q(bool=false) %!q(bool=false)}
```

Reply:
Oh thanks! That's a lot cleaner :D